game/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.22)
+project(game_of_life_exe)
+
+file(GLOB SOURCES "*cpp")
```

Comment:
I would propose consistently using either TARGET_SOURCES() or adding the sources via the add_<type>(name type sources) command rather than approaching it differently for the lib and exe.